### ISSUE TITLE
Some questions for setup

### ISSUE THREAD

**Issue by callum-tait-pbx** [summerwind/actions-runner-controller#75]
This looks like a super interesting project however I am unclear on a few things:

1. I've got cert-manager installed, but what do I actually need to do. I assume I need to install the webhook server cert as my pod is failing to launch because:

MountVolume.SetUp failed for volume "cert" : secret "webhook-server-cert" not found

2. There isn't anything documented about how you expect the pod to be exposed to github.com? What should I be doing to setup the webhook endpoint as required when creating the github app.

3. I'm surprised the apps client secret and client id aren't needed?

4. Any plans to support an webhook secret to further secure the webhook?

**Comment by summerwind** [comment]
The cert-manager is used to issue a certificate for the Validation webhook used by this controller and has nothing to do with GitHub webhooks.


> `MountVolume.SetUp failed for volume "cert" : secret "webhook-server-cert" not found`

This error probably means that the cert-manager was not able to issue the certificate correctly. What is the result of the following command?

```
$ kubectl get secrets webhook-server-cert -n actions-runner-system
```

**Comment by callum-tait-pbx** [comment]
I'm back! I've got cert-manager working. So if I run the above then I get `Error from server (NotFound): secrets "webhook-server-cert" not found` as you would expect as I never created a secret. I followed the README.md and there was never a step to add in that secret?

For our environment we have to prefix the namespaces with sre, so in my case the `cert-manager` is in the namespace `sre-cert-manager`. The `actions-runner-system` is in `sre-actions-runner-system`.
